frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -352,6 +353,24 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
             Note: Enabling monitoring will allow any operator or workload running on this namespace
             to contribute metrics to the cluster metric set.
           </small>
+          <br />
+          <br />
+          <Alert
+            isInline
+            className="co-alert co-alert--scrollable"
+            variant="warning"
+            title="Namespace monitoring"
+          >
+            Please note that installing non RedHat operators into openshift namespaces and enabling
+            monitoring voids user support. Enabling cluster monitoring for non RedHat operators can
+            lead to malicious metrics data overriding existing cluster metrics. For more
+            information, see the{' '}
+            <ExternalLink
+              href="https://docs.openshift.com/container-platform/4.2/monitoring/cluster-monitoring/configuring-the-monitoring-stack.html#maintenance-and-support_configuring-monitoring"
```

Comment:
Use our `openshiftHelpBase` (documentation.tsx)
```suggestion
              href=`{$openshiftHelpBase}monitoring/cluster-monitoring/configuring-the-monitoring-stack.html#maintenance-and-support_configuring-monitoring`}
```